src/components/FilmCardComponent/FilmCard.jsx

```diff
@@ -1,35 +1,76 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './FilmCard.module.css';
+import ApiClient from '../../api/apiClient';
 
-const FilmCard = (props) => {
-    return (
-        <div
-            className={`card ${styles.filmCard}`}
-            style={{ width: 18 + 'rem' }}
-        >
-            <img
-                src={'poster_path'}
-                className={`card-img-top ${styles.filmCardImg}`}
-                alt="poster"
-            />
-            <div className={`card-body ${styles.filmCardBody}`}>
-                <h5 className={`card-title ${styles.filmCardTitle}`}>
-                    {props.title}
-                </h5>
-                <p className={`card-text ${styles.filmCardYear}`}>
-                    {props.release_date}
-                </p>
-                <p className={`card-text ${styles.filmCardGenresList}`}></p>
+const api = new ApiClient();
+
+class FilmCard extends React.Component {
+    constructor(props) {
```

Comment:
it is an old version of defining state. Currently, you can write state = {...} instead of 
```
constructor(props) {super(props);
        this.state = {
            genres: null,
        };
    }
```

Reply:
Yeap, I know, but we dont have configurable babel. state = {...} dont work. We dont have to fix it

Reply:
As far as I know, it is working by default, isn't it?

Reply:
I dont know why, but now its working))) Before review, it is not.

Reply:
Fixed